products/apache-camel.md

```diff
@@ -27,12 +27,12 @@ auto:
 releases:
 -   releaseCycle: "3.22"
     lts: true
-    releaseDate: 2023-12-26
+    releaseDate: &id001 2023-12-19
```

Comment:
This is valid ([YAML anchor](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/)), but don't know which change triggered this behavior yet.

Reply:
Fixed in #269.